template_config.yml

```diff
@@ -28,6 +28,7 @@ flake8: true
 github_org: pulp
 issue_tracker: github
 keep_ci_update_for_latest_branches: 5
+keep_ci_update_since_branch: null
```

Comment:
@dralley which branch should we pick?

Reply:
3.14 is a lower bound, but 3.15 and 3.16 are dead branches.

Reply:
I'll make 3.14,
when pulpbot opens PR for 3.15 and 3.16 we just close them